vng_api_common/utils.py

```diff
@@ -90,3 +95,14 @@ def get_uuid_from_path(path: str) -> str:
     uuid.UUID(uuid_str)
 
     return uuid_str
+
+
+def request_object_attribute(url: str, attribute: str, resource: Union[str, None] = None) -> str:
+    Client = import_string(settings.ZDS_CLIENT_CLASS)
+    client = Client.from_url(url)
+    client.auth = APICredential.get_auth(url)
+    try:
+        result = client.retrieve(resource, url=url)[attribute]
+    except (ClientError, KeyError):
+        result = ''
```

Comment:
See https://github.com/VNG-Realisatie/gemma-zaakregistratiecomponent/pull/64#discussion_r285487310 :-)

Reply:
`logger.warning` is added to the function